src/hugolib/config.go

```diff
@@ -166,45 +168,59 @@ func LoadConfig(d ConfigSourceDescriptor, doWithConfig ...func(cfg config.Provid
 		}
 	}
 
+	const delim = "__env__delim"
+
 	// Apply environment overrides
 	if len(d.Environ) > 0 {
-		// Extract all that start with the HUGO_ prefix
-		const hugoEnvPrefix = "HUGO_"
-		var hugoEnv []string
+		// Extract all that start with the HUGO prefix.
+		// The delimiter is the following rune, usually "_".
+		const hugoEnvPrefix = "HUGO"
```

Comment:
Does this need to be changed as well?

Reply:
Not in v1. That would break backwards compatibility.